google-cloud-bigtable-stats/src/main/java/com/google/cloud/bigtable/stats/BuiltinViewConstants.java

```diff
@@ -59,6 +42,13 @@ class BuiltinViewConstants {
                   1.0, 2.0, 3.0, 4.0, 5.0, 6.0, 7.0, 8.0, 9.0, 10.0, 15.0, 20.0, 30.0, 40.0, 50.0,
                   100.0)));
 
+  private static final Aggregation PER_CONNECTION_ERROR_COUNT_AGGREGATION =
```

Comment:
This only goes up to 30k per connection, would that be enough? I would increase the upper bound